motivation.md

```diff
@@ -5,12 +5,15 @@ subtitle: Motivation
 ---
 ## Why Use Version Control?
 
-Let some software handle the versions of your project let you focus on your
-project.
+Using a software tool to handle the versions of your project files lets you focus on the more interesting/innovative aspects of your project
 
 ## Why Mercurial?
 
-FIXME
+* It's easy to set up
+* A few,
+  easy to remember commands are all you need for most day-to-day version control tasks
+* The [Bitbucket](https://bitbucket.org/) hosting service provides a web-based collaboration service
+* Every copy of a Mercurial repository is a full backup of a project and its history
```

Comment:
You could add (though that's not relevant for beginners):
* It's fast / performant
* It's implemented in Python and can (therefore) be used as a library